packages/globlon/globlon.1.0/opam

```diff
@@ -0,0 +1,36 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A globbing library for OCaml"
+description:
+  "A globbing library for OCaml. We use it to match file paths against patterns."
+maintainer: ["bleepbloopisfy"]
+authors: ["bleepbloopisfy"]
+license: "MIT"
+homepage: "https://github.com/bleepbloopisfy/globlon"
+bug-reports: "https://github.com/bleepbloopisfy/globlon/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/bleepbloopisfy/globlon.git"
```

Comment:
```suggestion
dev-repo: "git+https://github.com/bleepbloopsify/globlon.git"
```

Reply:
done, updated in my dune-project, and ran `opam publish` again